api/routes/books.js

```diff
@@ -26,53 +34,89 @@ router.get('/:id', async (req, res, next) => {
 
 // You should only be able to create a book if the user is an admin
 router.post('/', async (req, res, next) => {
-  const status = 200
-  try {
-    const book = await Book.create(req.body)
-    if (!book) throw new Error(`Request body failed: ${JSON.stringify(req.body)}`)
+  validToken = tokenValidate(req);
+  const user = await User.findOne({ _id: validToken.id })
+  
+  if (!user.admin == true) {
```

Comment:
I think this is the opposite of what you want. We only want the user to be able to create a book if they are an admin. So, I think we just need `if (user.admin)` here.